trans_tdsm.py

```diff
@@ -498,6 +502,8 @@ def main(config=None):
             save_name = os.path.join(training_file_path,'input_dists_transformed.png')
             fig.savefig(save_name)
 
+
+        train_model_name = "/afs/cern.ch/work/j/jthomasw/private/NTU/fast_sim/tdsm_encoder/training_20230830_1430_output/ckpt_tmp_499.pth" #Default model name 
```

Comment:
I'm don't think we want a default training file unless we are going to load it with a default config yaml so the parameter settings used are clear. Again, will leave this for now but something to keep in mind.